### PR TITLE
Fix grammar and styling in `Storyboard/Scripting/Variables`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by Walavouchey — COMMENTED** [review]
while these changes are ok, in terms of writing the most glaring issue is that the article contains a whole lot of "you". they can all be removed.

File: wiki/Storyboard/Scripting/Variables/en.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 # Storyboard scripting variables
 
-You can strings of text, up to a whole line, as **variables** for use elsewhere in an .osb file (note that the value can't change during the gameplay, so for those with software experience, think of it more like a constant). *.osu files do not support this.*
+**Variables** are abstract containers holding custom values that can be used elsewhere in an `.osb` file (note that the value cannot be changed during gameplay, so for those with programming experience, think of it more like a constant). `.osu` files *do not* support this.
```
Comment (The-Last-Cookie) [inline wiki/Storyboard/Scripting/Variables/en.md:3]: Tell me if I should remove the part in brackets too. I'm not sure if this is too informal.

Comment (Walavouchey) [reply wiki/Storyboard/Scripting/Variables/en.md:3]:
```suggestion
**Variables** are custom aliases for other values, typically long or common strings, which can be reused elsewhere in `.osb` files. These cannot be changed dynamically during gameplay, meaning they are constant values. They are *not supported* in `.osu` files.
```

File: wiki/Storyboard/Scripting/Variables/en.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 
 ## Usage
 
-The declaration of variables occurs in the separate \[Variables\] section of the .osb, located at the top of the file:
+The declaration of variables occurs in the separate \[Variables\] section located at the top of the `.osb` file:
```
Comment (Walavouchey) [inline wiki/Storyboard/Scripting/Variables/en.md:7]:
```suggestion
The declaration of variables occurs in a separate \[Variables\] section typically located at the top of the `.osb` file:
```

File: wiki/Storyboard/Scripting/Variables/en.md [hunk 3]
```diff
@@ -41,4 +41,4 @@ Sprite,Pass,Centre,"Sample.png",320,240
 _C,0,6000,7000,12,12,12
 ```
 
-Saving will replace the "12"s in colour specification with "$number_of_loops".
+Saving will replace the "12"s in colour specification with `$number_of_loops`.
```
Comment (Walavouchey) [inline wiki/Storyboard/Scripting/Variables/en.md:44]:
```suggestion
Saving will replace the "12"s in the colour specification with `$number_of_loops`.
```

File: wiki/Storyboard/Scripting/Variables/en.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
 
 ```
 [Variables]
-$color_link=0,255,0
+$colour_link=0,255,0
 $sample_path="Sample.png"
 ```
 
-You can use the variable in your code by typing the name (left hand side of the declaration, including the $) in your code. For instance, with the above declarations, this:
+Variables are used in the code by typing their name (left hand side of the declaration, including the $). For instance, with the above declarations, this:
```
Comment (Walavouchey) [inline wiki/Storyboard/Scripting/Variables/en.md:15]:
```suggestion
Variables are used in the code by typing their name (left-hand side of the declaration, including the $). For instance, with the above declarations, this:
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**